services/QuillLMS/client/app/bundles/Teacher/components/classrooms_with_students/ClassroomsWithStudents.jsx

```diff
@@ -16,37 +16,34 @@ export default class ClassroomsWithStudents extends React.Component {
   }
 
   classroomUpdates = () => {
-    const classrooms_data = [];
-    let classroomsWithNoAssignedStudents = 0;
-    this.props.classrooms.forEach((classy) => {
```

Comment:
It is insane to me that this is a real variable name in this function given that it reference a classroom object.